spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/core/SpannerTemplateTests.java

```diff
@@ -450,6 +451,19 @@ void findKeySetTestEagerOptions() {
     verify(this.databaseClient, times(1)).singleUse();
   }
 
+  @Test
+  void findKeySetTestEagerConvertedOptions() {
+    SpannerTemplate spyTemplate = spy(this.spannerTemplate);
+    KeySet keys = KeySet.newBuilder().addKey(Key.of("key1")).addKey(Key.of("key2")).build();
+    SpannerReadOptions options =
+            new SpannerReadOptions()
+                    .addReadOption(mock(Options.ReadAndQueryOption.class))
+                    .addReadOption(mock(Options.ReadQueryUpdateTransactionOption.class));
+    spyTemplate.read(ParentEntity.class, keys, options);
+    verify(spyTemplate, times(1)).read(eq(ParentEntity.class), same(keys), eq(options));
```

Comment:
nit: `times(1)` can be omitted, `verify(mock)` is alias to verify(mock, times(1)) 
```suggestion
    verify(spyTemplate).read(eq(ParentEntity.class), same(keys), eq(options));
```